tests/end_to_end/test_hdfs.sh

```diff
@@ -21,11 +21,37 @@
 . utils.sh
 
 eval $(parse_yaml $cluster_config "pai_")
-hdfs_uri=$pai_clusterinfo_restserverinfo_hdfs_uri
-webhdfs_uri="$(echo $hdfs_uri | sed -e "s/^hdfs/http/g" | sed -e "s/9000/50070/g")"
+hdfs_host=$pai_clusterinfo_hadoopinfo_hadoop_vip
+paifs_arg="--host $hdfs_host --port 50070 --user root"
```

Comment:
>50070 [](start = 36, length = 5)

configurable?

Reply:
- hdfs host is read from clusterconfig.yaml
- webhdfs port is the default port and also be read from clusterconfig.yaml, but our hadoop config uses the default port and can't config it

Reply:
user can manually change hadoop config after the prepare-hadoop step. but yeah we can do this later

---
In reply to: [161502491](https://github.com/Microsoft/pai/pull/195#discussion_r161502491) [](ancestors = 161502491)

Reply:
ok